pkg/app/web/src/mocks/services/insight.ts

```diff
@@ -28,6 +29,11 @@ export const getInsightApplicationCountHandler = createHandler<
   return response;
 });
 
+export const getInsightApplicationCountNotFound = createHandlerWithError(
```

Comment:
Where is this used?

Reply:
@nghialv 
Oh, I added it even though I haven't used it yet.
I'll remove it.

Reply:
👍